benchmarks/sindex.py

```diff
@@ -1,8 +1,14 @@
-from geopandas import read_file, datasets
-from geopandas.sindex import VALID_QUERY_PREDICATES
+from shapely.geometry import Point
 
+from geopandas import read_file, datasets, GeoSeries
+from geopandas.sindex import _get_sindex_class
+
+
+# Derive list of valid query predicates based on underlying index backend;
+# we have to create a non-empty instance of the index to get these
+index = _get_sindex_class()(GeoSeries([Point(0, 0)]).values.data)
```

Comment:
```suggestion
index = GeoSeries([Point(0, 0)]).sindex
```

Does that also work?